lib/pbench/server/database/models/server_config.py

```diff
@@ -62,18 +57,14 @@ def __str__(self) -> str:
 
 
 class ServerConfigMissingKey(ServerConfigError):
-    """
-    Attempt to set a ServerConfig with an empty key.
-    """
+    """Attempt to set a ServerConfig with an empty key."""
 
     def __str__(self) -> str:
         return "Missing key name"
```

Comment:
What's the difference between this exception (`ServerConfigMissingKey`) and the previous (`ServerConfigNullKey`), given that they both have the same docstring?...

I think either line 49 or line 60 should be modified so that a reader can easily understand the distinction.  (I realize, Peter, that you didn't provide this text, but your modification to it brought it to light, and you're in a position to correct the problem.)

Reply:
The `NullKey` variant is used when decoding a SQL null key constraint error, so the docstring could definitely be tweaked. I guess I was lazy. ☹️

Reply:
I'll tweak it, thanks for catching that.

Reply:
Fixed.